src/utils/overpass.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var dataHandler = require('./data_handler');
+var api = require('../config/api');
+
+var query = function() {
+  var request = dataHandler.getOverpassQuery();
+  var xhttp = new XMLHttpRequest();
+  xhttp.onreadystatechange = function() {
+    if (xhttp.readyState == 4) {
+      if (xhttp.status == 200) {
+        dataHandler.setOverpassOutput(JSON.parse(xhttp.response));
+        applyResponse();
```

Comment:
We could move the body of `applyResponse` here in order to use the Overpass result directly rather than storing it.